features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -1,310 +1,300 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-	Copyright (c) 2010-2019 Contributors to the openHAB project
-
-	See the NOTICE file(s) distributed with this work for additional
-	information.
-
-	This program and the accompanying materials are made available under the
-	terms of the Eclipse Public License 2.0 which is available at
-	http://www.eclipse.org/legal/epl-2.0
-
-	SPDX-License-Identifier: EPL-2.0
-
--->
+<!-- Copyright (c) 2010-2019 Contributors to the openHAB project See the NOTICE file(s) distributed with this work for additional
```

Comment:
It shouldn't format these XML headers.

Reply:
I fixed the headers by running `mvn license:format`.

@seime Do you have an idea how to exclude these parts from automatical formatting?

Reply:
@cweitkamp 
There is a generic option avoid joining comments that could be added to the xml formatter settings file
```
formatCommentJoinLines=false
```

Reply:
Thanks. I created https://github.com/openhab/static-code-analysis/pull/375 for it.